content/Hardware/Core Boards/Burn-the-bootloader-on-UNO-Mega-and-classic-Nano-using-another-Arduino.md

```diff
@@ -0,0 +1,181 @@
+    <td class="tg-29y7" style="border-color: #000000;border-style: solid;border-width: 1px;font-family: Arial, sans-serif;font-size: 14px;overflow: hidden;padding: 10px 5px;word-break: normal;background-color: #ff0000;color: #ffffff;text-align: center;vertical-align: top;">5V</td>
+    <td class="tg-ru72" style="border-color: #000000;border-style: solid;border-width: 1px;font-family: Arial, sans-serif;font-size: 14px;overflow: hidden;padding: 10px 5px;word-break: normal;background-color: #fe0000;color: #ffffff;text-align: center;vertical-align: top;"><span style="font-weight:400;font-style:normal">5V</span></td>
+  </tr>
+  <tr>
+    <td class="tg-5hz4" style="border-color: #000000;border-style: solid;border-width: 1px;font-family: Arial, sans-serif;font-size: 14px;overflow: hidden;padding: 10px 5px;word-break: normal;background-color: #330001;color: #ffffff;text-align: center;vertical-align: top;">GND</td>
+    <td class="tg-5hz4" style="border-color: #000000;border-style: solid;border-width: 1px;font-family: Arial, sans-serif;font-size: 14px;overflow: hidden;padding: 10px 5px;word-break: normal;background-color: #330001;color: #ffffff;text-align: center;vertical-align: top;"><span style="font-weight:400;font-style:normal">GND</span></td>
+  </tr>
+</tbody>
+</table>
+
+When the boards are connected, you can proceed with [burning the bootloader in Arduino IDE](#ide).
+
+<figure style="width: 500px; margin: 0;">
+    <img src="https://content.arduino.cc/assets/Nano_to_Uno_burn_bootloader.png" alt="Circuit Diagram. UNO and Nano connected with digital and power pins.">
+    <figcaption style="text-align: center; font-style: italic;">Example: Using an UNO to program a Nano (digital and power pins).</figcaption>
+</figure>
+
+<a id="icsp"></a>
+
+### Option B: Using the ICSP header pins
+
+![The ICSP header on Uno R3, Nano and Mega.](img/icsp-headers.png)_Note that the orientation of the header relative to the USB port is different on the Nano._
+
+1. Connect every programmer ICSP pin except RESET to the equivalent ICSP pin on the target.
+2. Connect PIN 10 on the programmer board to any RESET pin on the target.
+
+<figure style="width: 400px; margin: 0;">
+    <img src="https://content.arduino.cc/assets/Connections-mega2Uno.png" alt="Circuit Diagram. UNO and Mega connected with ICSP pins.">
+    <figcaption style="text-align: center; font-style: italic;">Example: Using an UNO to program a Mega (ICSP pins).</figcaption>
+</figure>
+
+---
+
+<a id="ide"></a>
+
+## Step 2: Burn the bootloader in Arduino IDE
+
+Now that the board connections are set up, we can use Arduino IDE to burn the bootloader.
+
+01. Open Arduino IDE.
+
+02. Connect the **programmer board** to the computer.
+
+03. Select your programmer board in _Tools > Board_, for example _Tools > Board > Arduino AVR Boards > Arduino Uno_.
+
+04. Select the port the board is connecting to in _Tools > Port_.
+
+05. Open _Tools > Memes > Examples_File > Examples > 11.ArduinoISP > ArduinoISP_.
```

Comment:
```suggestion
05. Open _File > Examples > 11.ArduinoISP > ArduinoISP_.
```
The path should be like this to match the image.

Reply:
handing in my resignation today